corehq/apps/dump_reload/sql/dump.py

```diff
@@ -229,6 +229,20 @@ class SqlDataDumper(DataDumper):
     slug = 'sql'
 
     def dump(self, output_stream):
+        """
+        When serializing data using JsonLinesSerializer().serialize(...), the additional parameters are set for
+        the following reasons:
+
+        use_natural_primary_keys is necessary for sharded models to ensure the primary key, which will not be
```

Comment:
Natural primary keys actually apply to _all_ sharded models, since all sharded models have the same issue with primary keys not being unique across shards. Does that explain this comment or is there still confusion here?

Reply:
No confusion - you can actually just ignore this suggestion